install_files_to_build.sh

```diff
@@ -5,7 +5,7 @@ cp -r resources build/
 
 mkdir -p build/bin/{win64,win32,osx64,osx32,linux64,linux32}
 
-rm -f build/bin/linux64/driver_openhmd.so && cp build/driver_openhmd.so build/bin/linux64 2> /dev/null && \
+rm -f build/bin/linux64/driver_openhmd.so && cp build/driver_openhmd.so build/bin/linux64/driver_openhmd.so 2> /dev/null && \
```

Comment:
So, this should only be a problem if build/bin/linux64 doesn't exist - which implies that the `mkdir` above didn't succeed for some reason. Perhaps we should add a success check to that line